tracer/test/Datadog.Trace.Tests/Configuration/ConfigurationSourceTests.cs

```diff
@@ -244,14 +247,17 @@ public void GlobalDefaultSetting(Func<GlobalSettings, object> settingGetter, obj
             Func<GlobalSettings, object> settingGetter,
             object expectedValue)
         {
+            IConfigurationSource source = new EnvironmentConfigurationSource();
+
             // save original value so we can restore later
             var originalValue = Environment.GetEnvironmentVariable(key);
+            Environment.SetEnvironmentVariable(key, string.Empty, EnvironmentVariableTarget.Process);
```

Comment:
This line seems unnecessary seeing as it's immediately changed on the next line?

Reply:
you're right, done in f55b55d